specification/devtestlabs/resource-manager/readme.md

```diff
@@ -110,5 +118,634 @@ See configuration in [readme.go.md](./readme.go.md)
 
 See configuration in [readme.java.md](./readme.java.md)
 
+## Suppression
 
-
+``` yaml
+directive:
```

Comment:
>directive [](http://example.com/codeflow?start=0&length=9)

since it's been a long time and  the original lint-diff error([https://github.com/Azure/azure-rest-api-specs/pull/17666/checks?sha=93f32e1fb5ea9dd065142f956f8543aa892de32f](https://github.com/Azure/azure-rest-api-specs/pull/17666/checks?sha=93f32e1fb5ea9dd065142f956f8543aa892de32f)) does not exist, could you kindly remove all the added suppressions and submit a new commit to create a new lint-diff result and then add the suppressions back. Then we can review whether they can be suppressed. Thanks.